pphysx3d/src/shapes/bounding_volume.rs

```diff
@@ -1,28 +1,32 @@
 use core::f32;
 
-use kiss3d::nalgebra::{self as na, Point, Point3, Vector2, Vector3};
+use kiss3d::nalgebra::{self as na, Point3, Vector3};
 
 pub trait BoundingVolume {
     fn interects(&self, other: &Self) -> bool;
     fn contains(&self, other: &Self) -> bool;
 }
 
 pub struct AABB {
-    mins: Point3<f32>, //two points is all it takes to represent a box
-    maxs: Point3<f32>,
+    //two points is all it takes to represent a box
+    pub mins: Point3<f32>,
+    pub maxs: Point3<f32>,
```

Comment:
```suggestion
    mins: Point3<f32>,
    maxs: Point3<f32>,
```
Probably shouldn't allow changing the AABB's fields directly